src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -1,26 +1,31 @@
 package com.minecolonies.client.gui;
 
+import com.blockout.Log;
 import com.blockout.controls.Button;
+import com.minecolonies.util.BlockUtils;
+import com.structures.helpers.Structure;
 import com.minecolonies.MineColonies;
-import com.minecolonies.colony.Schematics;
+import com.minecolonies.colony.Structures;
 import com.minecolonies.lib.Constants;
 import com.minecolonies.network.messages.BuildToolPlaceMessage;
-import com.minecolonies.util.BlockPosUtil;
 import com.minecolonies.util.LanguageHandler;
-import com.minecolonies.util.SchematicWrapper;
-import com.schematica.Settings;
-import com.schematica.client.renderer.RenderSchematic;
-import com.schematica.client.util.RotationHelper;
-import com.schematica.world.storage.Schematic;
+import com.structures.helpers.Settings;
 import net.minecraft.block.Block;
+import net.minecraft.client.Minecraft;
 import net.minecraft.entity.player.InventoryPlayer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.util.Mirror'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)